vars/pipelineDockerisedVega.groovy

```diff
@@ -392,7 +463,7 @@ Map<String,Closure> getPrepareVegatoolsStages(
                 }
             }
         }
-        stage('Build Vegatools Docker Image') {
+        /*stage('Build Vegatools Docker Image') {
```

Comment:
not building vegatools?

Reply:
Other images are required for DV. The `vegatools` is not needed for that.
Currently, the vegatools is used to convert checkpoint to JSON format. I plan to use it also to stream and logs all the events from the data-node and the vega core. But in separate ticket.

Reply:
ok